examples/sandbox/src/App.js

```diff
@@ -48,6 +48,7 @@ export default function App() {
             }
           }
         },
+        disjunctiveFacets: ["acres"],
```

Comment:
This was a small update to fix the sandbox application. The SingleSelectFacet needs a disjunctive facet to work properly.